src/components/modules/api/blocks.ts

```diff
@@ -303,16 +304,17 @@ export default class BlocksAPI extends Module {
    *
    * @param id - id of the block to update
    * @param data - the new data
+   * @param tunes - (optional) tune data
    */
-  public update = async (id: string, data: Partial<BlockToolData>): Promise<BlockAPIInterface> => {
+  public update = async (id: string, data: Partial<BlockToolData>, tunes?: {[name: string]: BlockTuneData}): Promise<BlockAPIInterface> => {
```

Comment:
What if I want to update only tunes? I think the data should be optional as well. 
One of "tune" or "data" should be provided.

Reply:
@neSpecc I have updated the code. I think it should just return provided block if neither new block data nor tunes data is provided in order to skip redundant and unnecessary update. What do you think about it?